packages/arb-token-bridge-ui/src/components/common/HeaderNetworkNotSupported.tsx

```diff
@@ -1,17 +0,0 @@
-import { InformationCircleIcon } from '@heroicons/react/outline'
```

Comment:
Oh damn, are we not using this anywhere? Let's not delete it, I'll write down a ticket for us to bring it back.

Reply:
No, I restored the file